plugins/inputs/ipset/ipset.go

```diff
@@ -46,8 +46,8 @@ func (ipset *Ipset) SampleConfig() string {
 `
 }
 
-func (ips *Ipset) Gather(acc telegraf.Accumulator) error {
-	out, e := ips.lister(ips.Timeout, ips.UseSudo)
+func (ipset *Ipset) Gather(acc telegraf.Accumulator) error {
```

Comment:
Here we diverge from the usual one-letter variable name... Only because it's i? :-)

Reply:
`rule.receiver-naming` requires that receiver names should be consistent (have the same var-name) and they weren't in this case. Previous receivers had used `ipset` so I adjusted,  but I can change all of them to `i` if you insist ;)

Reply:
Again I'm not insisting, but if you don't need to add an exception, I wouldn't add one. Just my opinion. :-)

Reply:
@srebhan changed to `i`

Reply:
docs seem to agree. resolving.